lib/workers/repository/onboarding/branch/index.spec.ts

```diff
@@ -390,23 +416,17 @@ describe('workers/repository/onboarding/branch/index', () => {
       it('detects missing rebase checkbox', async () => {
         const pr = { bodyStruct: { rebaseRequested: undefined } };
         platform.getBranchPr.mockResolvedValueOnce(mock<Pr>(pr));
-
         await checkOnboardingBranch(config);
-
         expect(logger.debug).toHaveBeenCalledWith(
           `No rebase checkbox was found in the onboarding PR`
         );
         expect(OnboardingState.prUpdateRequested).toBeTrue();
-        expect(git.mergeBranch).toHaveBeenCalledOnce();
-        expect(scm.commitAndPush).toHaveBeenCalledTimes(0);
```

Comment:
why removed?

Reply:
These assertions are extra. We have already checkd with the logger statement that checkbox isn't present.